File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.cs
using System;

public class Solution {
    public int solution(int a, int b) {        
        string s = "";
        int sum = 2 * a * b;
        
        s = a.ToString() + b.ToString();
        
        if(sum > int.Parse(s)) return sum;
        else return int.Parse(s);
    }
}

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.cpp
#include <string>
#include <vector>
#include <bits/stdc++.h>

using namespace std;

int solution(int a, int b) {
    int a1 = stoi(to_string(a) + to_string(b));
    int a2 = a * b * 2;
    return a1 > a2 ? a1 : a2;
}